myuw/static/vue/store/hx_toolkit.js

```diff
@@ -0,0 +1,24 @@
+import {fetchBuilder, extractData, buildWith} from './model_builder';
+
+const postProccess = (response) => {
+    const html = response.data;
+    const parser = new DOMParser();
+    var data = {};
+
+    const htmlDoc = parser.parseFromString(
+        html, 'text/html'
+    );
+
+    if(htmlDoc.links[0] !== undefined) {
+        data.article_html = htmlDoc.links[0].outerHTML;
+    }
+    
+    return data;
+}
+const customActions = {
+    fetch: fetchBuilder('/api/v1/hx_toolkit/week/', postProccess, 'text'),
```

Comment:
as done as https://github.com/uw-it-aca/myuw/blob/ba022783306e6b711dac11bb39a593f67ff9b16f/myuw/static/js/ws_data.js#L1347-L1376, week should be passed as a url param. Example https://github.com/uw-it-aca/myuw/blob/feature/vue/myuw/static/vue/components/cards/grades.vue#L188
and 
https://github.com/uw-it-aca/myuw/blob/feature/vue/myuw/static/vue/store/courses.js#L10